x-pack/plugins/monitoring/public/plugin.ts

```diff
@@ -89,7 +88,17 @@ export class MonitoringPlugin
         this.overrideAlertingEmailDefaults(deps);
 
         const monitoringApp = new AngularApp(deps);
-        return monitoringApp.destroy;
+        const removeHistoryListener = params.history.listen(location => {
+          if (location.pathname === '' && location.hash === '') {
+            params.history.push({ hash: uiRoutes.redirect?.redirectTo || '/no-data' });
```

Comment:
I think you'll want to use `replace` instead of `push` because `push` will retain a entry in the browser history so that pressing the back button will not go back to the previous location.

Reply:
Ah good catch 👍

Treating out destination page (from the main landing) as a redirect did the trick